### PR TITLE
DEMO-35: Improve template organization

### PR DESCRIPTION
**Pull request by nicolas-bastien**
Hi @clash82,

In this PR I did : 

- move layout subtemplates to a layout folder
- fix hardcoded comment code
- fix hardcoded copyright code
- move javascript call to the end of the page

https://jira.ez.no/browse/DEMO-35

### CODE REVIEW

**Comment by nicolas-bastien** [comment]
@clash82 commit rewritten with Jira prefix

Are you ok with this one now ?

File: app/Resources/views/full/contact_form.html.twig [hunk 1]
```diff
@@ -73,22 +73,24 @@
             </div>
         </div>
     </section>
+{% endblock %}
 
+{% block javascripts %}
     <script src="http://maps.googleapis.com/maps/api/js"></script>
-    <script>
+    <script type="text/javascript">
```
Comment (clash82) [inline app/Resources/views/full/contact_form.html.twig]: This is HTML5, we don't need `type="text/javascript"` anymore (it is used by default).

File: app/Resources/views/layout/footer.html.twig [hunk 2]
```diff
@@ -26,7 +26,7 @@
             <div class="row">
                 <div class="col-xs-offset-1 col-xs-5 space-top-half-more space-bottom-less">
                     <ul>
-                        <li class="white small-extra">Copyright &copy; 2015 eZ Systems AS (except where otherwise noted). All rights reserved. <a href="//ez.no/privacy-policy">Privacy Policy.</a> Powered by the <a href="//ez.no/ezpublish"> eZ Platform</a></li>
+                        <li class="white small-extra">Copyright &copy; {{ "now"|date("Y") }} eZ Systems AS (except where otherwise noted). All rights reserved. <a href="//ez.no/privacy-policy">Privacy Policy.</a> Powered by the <a href="//ez.no/ezpublish"> eZ Platform</a></li>
```
Comment (clash82) [inline app/Resources/views/layout/footer.html.twig]: By default we're using single quotes in Twig helpers in this project ;)

**Comment by clash82** [comment]
@damianz5 can you have a look?

File: app/Resources/views/full/contact_form.html.twig (hunk 1 above)
Comment (nicolas-bastien) [inline app/Resources/views/full/contact_form.html.twig]: Done

File: app/Resources/views/layout/footer.html.twig (hunk 2 above)
Comment (nicolas-bastien) [inline app/Resources/views/layout/footer.html.twig]: Done

File: app/Resources/views/pagelayout_scripts.html.twig [hunk 3]
```diff
@@ -1,22 +0,0 @@
...
-
-<script type="text/javascript">
-    {% autoescape 'js' %}
-        {# TODO: move hardcoded identifier to configuration #}
-        var disqus_shortname = 'ezpublishplatformdemo';
-
-        (function () {
-            var s = document.createElement('script'); s.async = true;
-            s.type = 'text/javascript';
-            s.src = '//' + disqus_shortname + '.disqus.com/count.js';
```
Comment (damianz5) [inline app/Resources/views/pagelayout_scripts.html.twig]: Are you sure it can be deleted?

Comment (nicolas-bastien) [inline app/Resources/views/pagelayout_scripts.html.twig]: ok I'll try to see if we cannot add that in the bundle so

Comment (nicolas-bastien) [inline app/Resources/views/pagelayout_scripts.html.twig]: https://github.com/ezsystems/CommentsBundle/pull/24

File: app/Resources/views/layout/footer.html.twig [hunk 4]
```diff
@@ -6,7 +6,7 @@
                     <a href="/"><img src="{{ asset('assets/images/meadow_white.png') }}" class="center-block img-responsive meadow-logo-bottom" /></a>
                 </div>
                 <div class="col-md-offset-1 col-md-3 space-top-half space-bottom-less">
-                    <h3 class="white">Meadow US Inc.</h3>
+                    <h3 class="white">Meadow US, Inc.</h3>
```
Comment (nicolas-bastien) [inline app/Resources/views/layout/footer.html.twig]: @DavidLiedle I updated my PR here to take your modification

File: app/Resources/views/layout/footer.html.twig [hunk 5]
```diff
@@ -26,7 +26,7 @@
             <div class="row">
                 <div class="col-xs-offset-1 col-xs-5 space-top-half-more space-bottom-less">
                     <ul>
-                        <li class="white small-extra">Copyright &copy; 2015 eZ Systems AS (except where otherwise noted). All rights reserved. <a href="//ez.no/privacy-policy">Privacy Policy.</a> Powered by the <a href="//ez.no/ezpublish"> eZ Platform</a></li>
+                        <li class="white small-extra">Copyright &copy; {{ 'now'|date('Y') }} eZ Systems AS (except where otherwise noted). All rights reserved. <a href="//ez.no/privacy-policy">Privacy Policy.</a> Powered by the <a href="//ez.no/ezpublish"> eZ Platform</a></li>
```
Comment (clash82) [inline app/Resources/views/layout/footer.html.twig]: `//ez.no/ezpublish` should be replaced by `//ezplatform.com`

**Comment by nicolas-bastien** [comment]
@clash82 done

**Pull request merged**